packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.tsx

```diff
@@ -594,8 +615,12 @@ class ViewportPositioner extends Foundation<
             }
         });
 
-        this.scrollTop = this.getViewportElement().scrollTop;
-        this.scrollLeft = this.getViewportElement().scrollLeft;
+        const viewPortElement: HTMLElement | null = this.getViewportElement();
+
+        if (!isNil(viewPortElement)) {
+            this.scrollTop = this.getViewportElement().scrollTop;
+            this.scrollLeft = this.getViewportElement().scrollLeft;
```

Comment:
should these just be `viewPortElement.scrollTop` etc since you're alerady storing the result of viewPortElement?

Reply:
yep